chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,611 @@
+Should any datasets be possible to merge one-to-one?
+And if so, will the have the same unique identifier?
+Should any datasets be possible to merge many-to-one (for example school and student)?
+And if so, what identifier will be used?
+Solving all of these questions before you acquire any data,
+and making sure that the full research team knows where to find this info,
+will save you a ton of time later on.
+
+Other characteristics should be updated as the project progresses.
+Example of those characteristics are:
+Where will the data be obtain from?
+Where in the project folder are the raw original data stored
+and where are back-ups of the raw data stored?
+See the wiki article linked to above for an template that you can use for your project.
+
+If you have multiple datasets with the same unit of observation
+then you should create a \textbf{master dataset}\sidenote{
+	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}}
+for each of those units of observation and include that in the required datasets.
+The master dataset should include an ID variable
+that can be used to link or merge all datasets with that unit of observation.
+It should also list project wide time-invariant information
+like sampling and treatment status.
+Additionally, any unit of observation
+that will be used in sampling or treatment assignment
+should have a master dataset,
+and that dataset should be used when sampling or assigning treatment.
+
+\section{Taking research design into account when planing your data work}
+
+The most important steps in data work planning
+to correctly translate a research design into data work
+are sampling and treatment assignment.
+Therefore, understanding and correctly implementing sampling and treatment assignment
+is essential to ensuring that data is acquired properly and that research
+is correctly carried out in the field, so that the results
+are valid interpretations of your research question.
+
+Not all projects need to do sampling and treatment assignment.
+Sampling is almost always required when you collect your own data,
+as it is often prohibitively expensive to collect data on all individuals.
+However, if the data you use is, for example,
+the tax filings of all companies in a country,
+then you can estimate the treatment effect
+using the full population and not a sample of it.
+Such big data might need a cluster of computers to process,
+so you might still want to know how make a representative sample,
+so that you can develop your code on a local computer
+before running it on an expensive computer cluster.
+
+Treatment assignment is only applicable if your research design is experimental,
+and the research team can control which individuals receive the treatment.
+While you might not yet have a population dataset to sample from,
+or a complete list of eligible observation to randomize assignment on,
+this is still the time to understand those tasks
+and what you need to think about as you acquire those datasets.
```

Comment:
flip to positive perspective. instead of what they might not have, what data is needed in order to successfully implement these random processes (acknowledging that such data might not be available yet, and that the next chapter covers data acquisition)